src/software/amazon/ion/impl/LocalSymbolTable.java

```diff
@@ -327,7 +327,7 @@ private LocalSymbolTable(LocalSymbolTable other, int maxId)
                     // As per the Spec, other field types are treated as
                     // empty lists
                     if(foundLocalSymbolList){
-                        throw new IonException("Multiple symbol tables found.");
+                        throw new IonException("Multiple symbol fields found within a single local symbol table.");
```

Comment:
These error messages really need to include location information.  Humans are going to want to debug this stuff, and we need to provide them with all relevant context at hand.